src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -54,12 +61,32 @@ public void parseCommand_delete() throws Exception {
     }
 
     @Test
-    public void parseCommand_edit() throws Exception {
+    public void parseCommand_book() throws Exception {
```

Comment:
For these tests, should we consider using more specific exceptions in the method header instead of "Exception"?